spec/services/course/duplication/object_duplication_service_spec.rb

```diff
@@ -341,19 +341,19 @@
         end
         let!(:content) { create(:course_material, folder: folder) }
 
-        context 'when folders are duplicated but not their parents' do
+        context 'when folder is duplicated but not its parents nor materials' do
```

Comment:
For the test structure, I'm not sure if all the cases are covered. Would suggest to approach it in a more structured way, ie.
```
context 'when materials and folders are present' do
  context 'when no folders are selected for duplication' do
    it 'dups the selected materials into root folder' {}
  end

  context 'when folders are selected for duplication' do
    context 'when folders parents are selected for duplication' { it 'assigns the parent folder correctly' {} }
    context 'when folders parent are not selected for duplication' { it 'sets parent to root folder of target course' {} }
    context 'when folder materials are selected for duplication ' { it 'assigns the folders of materials correctly' {} }
  end
end
```